core/eth/contracts/types.go

```diff
@@ -0,0 +1,30 @@
+package contracts
+
+import (
+	"chainlink/core/eth"
+)
+
+type MaybeDecodedLog struct {
```

Comment:
was this stuff moved verbatim as well?

Reply:
No, these are just little wrapper types that I spun up temporarily.  It's probably worth mentioning that these will disappear in https://github.com/smartcontractkit/chainlink/pull/2458 which creates a more composable way of dealing with logs.